recipe/meta.yaml

```diff
@@ -11,17 +11,17 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: true  # [py2k]
+  skip: true  # [py2k and py34 and win]
```

Comment:
I think maybe 'or' would be the correct operator here

Reply:
You're absolutely right.
Correction made.